pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "netutils"
-version = "1.4.1"
+version = "1.4.2"
```

Comment:
You can remove this, we will take care of it and being that it is a new feature, it would go into 1.5.0.

Reply:
This was to test getting this to deploy in a customer environment. I'll remove it before the final push.